ios/Video/RCTDecoderPropertiesModule.m

```diff
@@ -0,0 +1,13 @@
+#import "RCTDecoderPropertiesModule.h"
+
+@implementation RCTDecoderPropertiesModule
+
+RCT_EXPORT_MODULE(RNVDecoderPropertiesModule);
+
+RCT_EXPORT_METHOD(getWidevineLevel : (RCTPromiseResolveBlock)resolve reject : (RCTPromiseRejectBlock)reject) {}
```

Comment:
I think you should al least resolve or reject the Promise... (maybe reject with an error "not implemented")

Reply:
These functions are meaningless but, acceptable.
I think we should throw an error to notify library contributors.
(we should wrapping decoder properties module on JS side)